### ISSUE TITLE
url not defined in express.js

### ISSUE THREAD

**Issue by tcoats** [odojs/odo#7]
```
ReferenceError: url is not defined
  at /home/identity-management/node_modules/odo/express.js:113:19
```

url needs to be added to the requirejs `define` statement before it can be used in the CORS section as `url.parse`

**Comment by ajnisbet** [comment]
Ah, I have that running locally, not sure how I left it out of the commit, will fix

**Comment by ajnisbet** [comment]
Fixed by 3f789f3ca
